lib/src/utils/constants.dart

```diff
@@ -0,0 +1,7 @@
+const int harden = 0x80000000;
+const int claCardano = 0xD7; // CLA for Cardano
+const int insDeriveAddress = 0x11; // INS for Derive Address
+const int p1ReturnAddressToHost = 0x01; // P1: request type for returning address to host
+const int p2Unused = 0x00; // P2: unused
```

Comment:
In quite a few places for `SendOperation` you manually write `p2: 0` or `p2: 0x00`. Would be better to use this constant instead.

Reply:
done